orchagent/fdborch.cpp

```diff
@@ -34,21 +34,66 @@ void FdbOrch::update(sai_fdb_event_t type, const sai_fdb_entry_t* entry, sai_obj
 
         (void)m_entries.insert(update.entry);
         SWSS_LOG_DEBUG("FdbOrch notification: mac %s was inserted into vlan %d", update.entry.mac.to_string().c_str(), entry->vlan_id);
+        
+        for (auto observer: m_observers)
+        {
+            observer->update(SUBJECT_TYPE_FDB_CHANGE, &update);
+        }
```

Comment:
this is about the fdb flush, why touch both LEARNED, AGED, MOVE events?

Reply:
separate the handling FLUSH from AGED and MOVE, at the same time remove unnecessary type cast.  No impaction to the handling of other events.